lib/perl/Genome/Model/ReferenceAlignment/Report/Summary.pm

```diff
@@ -364,10 +364,13 @@ sub _generate_summary_information
     unless (@lq_files) {
         @lq_files = $build->get_variant_bed_file('snps_all_sequences');
     }
-    my $lq_snp_calls = `wc -l @lq_files | tail -n 1`;
-    $lq_snp_calls =~ s/\s\S+\s*$//i;
-    $lq_snp_calls =~ s/\s//g;
-    my $unfiltered_snp_calls = $filtered_snp_calls + $lq_snp_calls;
+    my ( $lq_snp_calls, $unfiltered_snp_calls );
+    if (@lq_files) {
+        $lq_snp_calls = `wc -l @lq_files | tail -n 1`;
+        $lq_snp_calls =~ s/\s\S+\s*$//i;
+        $lq_snp_calls =~ s/\s//g;
+        $unfiltered_snp_calls = $filtered_snp_calls + $lq_snp_calls;
```

Comment:
`$unfiltered_snp_calls` should probably be initialized to `$filtered_snp_calls` rather than being left undefined when there are no `@lq_files`.  (Then this could be a `+=`.)